### PR TITLE
Grafana-UI: Removes knobs still showing in storybook stories

### PR DESCRIPTION
**Pull request by thisisobate**
**What this PR does / why we need it**:
This PR completely disables knobs from the storybook addons pane.
**Which issue(s) this PR fixes**:

Relates to #34417

**Special notes for your reviewer**:
A4 - 1

### CODE REVIEW

**Review by hugohaggmark — COMMENTED** [review]
I don't have much context so sorry for asking an obvious/stupid question but does this disable knobs for all stories? If so, is there a reason why we don't remove knobs completely?

**Comment by thisisobate** [comment]
> does this disable knobs for all stories?

Yes, it does.

> is there a reason why we don't remove knobs completely?

Yes. The reason is that there are still some component stories and storybook utility functions that use the addon knobs- would fix those in a separate PR by migrating them to use controls instead. Disabling/removing the knobs do not break the storybook so I feel it makes sense for us to do so. @hugohaggmark

File: packages/grafana-ui/.storybook/preview.ts [hunk 1]
```diff
@@ -58,5 +58,6 @@ export const parameters = {
   },
   knobs: {
     escapeHTML: false,
```
Comment (hugohaggmark) [inline packages/grafana-ui/.storybook/preview.ts:60]: Maybe remove this if we disable knobs?
```suggestion
```

**Review by hugohaggmark — APPROVED** [review]
LGTM but I haven't tested this locally 🎉

File: packages/grafana-ui/.storybook/preview.ts (hunk 1 above)
Comment (thisisobate) [reply packages/grafana-ui/.storybook/preview.ts:60]: sure...will do. Thanks!

**Comment by hugohaggmark** [comment]
> @hugohaggmark I think it also makes sense for us to remove the `knobs disable` config in individual stories since we're already doing that in the preview.ts file which affects the entire stories. Would do that in a new commit.

Great catch 👍

**Pull request merged**